buildattr64/buildattr64.rst

```diff
@@ -0,0 +1,901 @@
+
+Terms and Abbreviations
+-----------------------
+
+The ABI for the Arm 64-bit Architecture uses the following terms and abbreviations.
+
+A32
+   The instruction set named Arm in the Armv7 architecture; A32 uses 32-bit
+   fixed-length instructions.
+
+A64
+   The instruction set available when in AArch64 state.
+
+AAPCS64
+   Procedure Call Standard for the Arm 64-bit Architecture (AArch64)
+
+AArch32
+   The 32-bit general-purpose register width state of the Armv8 architecture,
+   broadly compatible with the Armv7-A architecture.
+
+AArch64
+   The 64-bit general-purpose register width state of the Armv8 architecture.
+
+ABI
+   Application Binary Interface:
+
+   1. The specifications to which an executable must conform in order to
+      execute in a specific execution environment. For example, the
+      *Linux ABI for the Arm Architecture*.
+
+   2. A particular aspect of the specifications to which independently produced
+      relocatable files must conform in order to be statically linkable and
+      executable.  For example, the CPPABI64_, AAELF64_, ...
+
+Arm-based
+   ... based on the Arm architecture ...
+
+Floating point
+   Depending on context floating point means or qualifies: (a) floating-point
+   arithmetic conforming to IEEE 754 2008; (b) the Armv8 floating point
+   instruction set; (c) the register set shared by (b) and the Armv8 SIMD
+   instruction set.
+
+Q-o-I
+   Quality of Implementation – a quality, behavior, functionality, or
+   mechanism not required by this standard, but which might be provided
+   by systems conforming to it.  Q-o-I is often used to describe the
+   toolchain-specific means by which a standard requirement is met.
+
+SIMD
+   Single Instruction Multiple Data – A term denoting or qualifying:
+   (a) processing several data items in parallel under the control of one
+   instruction; (b) the Arm v8 SIMD instruction set: (c) the register set
+   shared by (b) and the Armv8 floating point instruction set.
+
+SIMD and floating point
+   The Arm architecture’s SIMD and Floating Point architecture comprising
+   the floating point instruction set, the SIMD instruction set and the
+   register set shared by them.
+
+SVE
+   The Arm architecture's Scalable Vector Extension.
+
+T32
+   The instruction set named Thumb in the Armv7 architecture; T32 uses
+   16-bit and 32-bit instructions.
+
+VG
+   The number of 64-bit “vector granules” in an SVE vector; in other words,
+   the number of bits in an SVE vector register divided by 64.
+
+ILP32
+   SysV-like data model where int, long int and pointer are 32-bit
+
+LP64
+   SysV-like data model where int is 32-bit, but long int and
+   pointer are 64-bit.
+
+LLP64
+   Windows-like data model where int and long int are 32-bit, but
+   long long int and pointer are 64-bit.
+
+
+This document uses the following terms and abbreviations.
+
+Link-unit
+   An executable or shared library. Also known as loadable-unit in
+   this document.
+
+Loadable-unit
```

Comment:
The existing docs don't use the term "loadable-unit". "link unit" has been used in several docs. Is there a rationale  to introduce this alias?

Reply:
In some internal feedback some of my colleagues didn't like link-unit :-) . Loadable unit was felt to be more inituitively understandable than link-unit.

I'm confortable with link-unit, but that's many years of familiarity.